src/Mono.Android/Xamarin.Android.Net/AndroidClientHandler.cs

```diff
@@ -58,7 +58,7 @@ namespace Xamarin.Android.Net
 	/// if they use a certificate with a fully validated trust chain) unless you store the CA certificates from your Android system in <see cref="TrustedCerts"/> along with
 	/// the self-signed certificate(s).</para>
 	/// </remarks>
-	public class AndroidClientHandler : HttpClientHandler
+	public sealed class AndroidClientHandler : HttpClientHandler
```

Comment:
Thanks @simonrozsival for the PR, but I'm afraid it's not enough just to make the class sealed :( If you read the docstring above the class, it talks about a method to implement SSL certificate validation by deriving a class from `AndroidClientHandler`.  Could you make changes to address that scenario, also updating the docs as needed? Thanks!